Sources/SWBTaskExecution/TaskActions/GenericCachingTaskAction.swift

```diff
@@ -320,11 +326,12 @@ fileprivate struct CacheKey {
             throw StubError.error("Could not load cached value")
         }
         let deserializer = MsgPackDeserializer(casObject.data)
-        try deserializer.beginAggregate(2)
+        try deserializer.beginAggregate(4)
```

Comment:
Huh, what happened here? store went from 3 to 4 but load went from 2 to 4. Was this just completely broken before?

Reply:
It was a bug